tests/Unit/Helpers/Domain/Creators/TestHelpers.hpp

```diff
@@ -221,4 +223,20 @@ void test_functions_of_time(
                }
              });
 }
+
+// Only checks that grid anchors exist. Doesn't check what they actually are
+// (because they can be anywhere in the domain)
+template <size_t Dim>
+void test_grid_anchors(const DomainCreator<Dim>& creator,
```

Comment:
Just delete this and check the grid anchors explicitly like you do for the two BCO domains (same as excision spheres). The other domains can either check nothing (since grid anchors aren't implemented yet) or check the grid anchors are `empty()`.